gubernator/view_pr_test.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+
+from webapp2_extras import securecookie
+
+import gcs_async_test
+import github.models as ghm
```

Comment:
Any reason not do do from github import models here?

a) `from whatever import foo` is preferred
b) models doesn't seem long/inconvenient enough to warrant an `import as`

https://google.github.io/styleguide/pyguide.html?showone=Packages#Packages